SCons/Tool/__init__.py

```diff
@@ -707,11 +715,6 @@ def __call__(self, env, libnode, **kw):
 
         return soname
 
-
-ShLibSonameGenerator = _LibSonameGenerator('ShLib')
-LdModSonameGenerator = _LibSonameGenerator('LdMod')
```

Comment:
Why remove these?

Reply:
These were no longer used so i removed them. However I reverted the changes to the D tools so these are no longer removed in d9c4217.